transformers/tokenization_utils.py

```diff
@@ -1142,7 +1142,7 @@ def convert_ids_to_tokens(self, ids, skip_special_tokens=False):
             if index in self.added_tokens_decoder:
```

Comment:
We should probably convert `index` to `int` before that so we can also index `self.added_tokens_decoder` with `index`.

Reply:
I don't think there's a need to cast `index` to `int` to index `self.added_tokens_decoder` as if `index` was taken from a `np.array`, then it is of type `int64` which can be used to index a dictionary. For example, before the fix, this crashes:

```py
from transformers import AlbertTokenizer
import numpy as np

tokenizer = AlbertTokenizer.from_pretrained("albert-base-v1")
tokenizer.decode(np.array([1,2,3]))

# TypeError: in method 'SentencePieceProcessor_IdToPiece', argument 2 of type 'int'
```

But this does not:

```py
from transformers import AlbertTokenizer
import numpy as np

tokenizer = AlbertTokenizer.from_pretrained("albert-base-v1")

tokenizer.add_tokens(["new_token"])
print(tokenizer.added_tokens_decoder)  # {30000: 'new_token'}

index = np.array([30000])[0]
print(type(index))  # <class 'numpy.int64'>

print(tokenizer.added_tokens_decoder[index])  # new_token
```